packages/react/src/patterns/blocks/ContentGroupHorizontal/ContentGroupHorizontal.js

```diff
@@ -52,13 +52,22 @@ ContentGroupHorizontal.propTypes = {
       eyebrow: PropTypes.string,
       heading: PropTypes.string.isRequired,
       copy: PropTypes.string.isRequired,
-      cta: PropTypes.arrayOf(
-        PropTypes.shape({
-          type: PropTypes.oneOf(['local', 'external']).isRequired,
-          href: PropTypes.string.isRequired,
-          copy: PropTypes.string.isRequired,
-        })
-      ).isRequired,
+      cta: PropTypes.shape({
+        heading: PropTypes.string,
+        items: PropTypes.arrayOf(
+          PropTypes.shape({
+            style: PropTypes.oneOf(['text', 'card', 'button', 'feature']),
```

Comment:
@asudoh I believe only `text` and `card` are currently supported.

```suggestion
            style: PropTypes.oneOf(['text', 'card']),
```

Reply:
Great catch - Found that [`style` there is set from another prop](https://github.com/carbon-design-system/ibm-dotcom-library/blob/v1.8.0/packages/react/src/components/LinkList/LinkList.js#L42), so removed this prop from the shape.